docs/components/home/Homepage.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <main class="main">
+    <article class="container border-red">
+      <div class="intro">
+        <p>
+          The documentation concerning
+          <br>
+          the npm modules of
+        </p>
+        <h1 class="title">
+          Shiki-Twoslash
+        </h1>
+        <p>
+          In which markdown code samples are powered by
+          <br>
+          the syntax engine of Visual Studio Code
+          <br>
+          mixed with the TypeScript compiler’s information
+        </p>
+
+        <div style="margin-top: 3rem; margin-bottom:3rem">
+          <img style="margin: 0 auto" src="/svgs/squiggle.svg" alt="Decoration" width="70" height="25.5">
```

Comment:
```suggestion
          <img class="dark:invert" style="margin: 0 auto" src="/svgs/squiggle.svg" alt="Decoration" width="70" height="25.5">
```

Little trick to support dark mode :P